cmake/FindCustomLapack.cmake

```diff
@@ -0,0 +1,128 @@
+# Distributed under the OSI-approved BSD 3-Clause License.
```

Comment:
The project itself is licensed under BSD-2-Clause, any reason to have an additional BSD-3-Clause license on the cmake files?

Reply:
I just took the standard license I found in most CMake finder module. But you're right, I'll downgrade it to the 2-clause one.